bluemix/terminal/table.go

```diff
@@ -157,3 +159,15 @@ func (t *PrintableTable) PrintJson() {
 	// mimic behavior of Print()
 	t.rows = [][]string{}
 }
+
+func (t *PrintableTable) PrintCsv() {
+	csvwriter := csv.NewWriter(t.writer)
+	err := csvwriter.Write(t.headers)
+	if err != nil {
+		fmt.Fprintln(t.writer, "Failed, header could not convert to csv format")
```

Comment:
@edsonarios the only thing I will add here is that these error messages are not translated. Does csvwriter not return a suitable error message that can be output or even returned? Generally I would expect the operation to fail if you get errors on these calls to Write and WriteAll -- `err` just is returned and the logic does not continue.

Reply:
Was added translation for message error, also was returner the error when fail the functions Write and WriteAll with the personalized message